.github/workflows/review-release-notes.yml

```diff
@@ -0,0 +1,37 @@
+name: Review Release Notes
+on: pull_request
```

Comment:
dont we need `push` here?

Reply:
I think on pull request its enough, (it will atomatically run after every push in case theres an open pull_request, it will not run in case you pushed and didnt open a pr)